mirrord/intproxy/src/proxies/incoming/interceptor.rs

```diff
@@ -258,6 +257,16 @@ impl HttpConnection {
                         .await
                         .map(HttpResponseFallback::Fallback)
                     }
+                    HttpRequestFallback::Chunked(_, _) => {
+                        HttpResponse::<Vec<u8>>::from_hyper_response(
```

Comment:
I think we want to use the `HttpResponse<InternalHttpBody>` type (it's preferred over `HttpResponse<Vec<u8>>` and we know it's supported by the agent - we got a chunked request)